lib/vrl/tests/tests/expressions/comparison/ge.vrl

```diff
@@ -19,4 +19,5 @@
 
     # fallible
     null >= 1 ?? "nope",
+    to_string([.foo]) >= "bar" ?? "nope",
```

Comment:
We'll also want to fill out the matrix of all other possibilities.

E.g.

```
"a" > "b"
"a" > "a"
"b" > "a"
```

And do that for all comparison operators.

It's a bit of an annoyance to have to write those out manually, especially for the comparison expression, but it's good to cover our bases. (and also, as @leebenson suggested, it's also good to test a few more, such as `1 > "a"` etc).